modules/adagioBidAdapter.js

```diff
@@ -692,15 +697,21 @@ export const spec = {
       return false;
     }
 
-    const { organizationId, site, placement } = params;
-    const adUnitElementId = params.adUnitElementId || internal.autoDetectAdUnitElementId(adUnitCode);
+    const { organizationId, site } = params;
+    const adUnitElementId = (params.useAdUnitCodeAsAdUnitElementId === true)
+      ? adUnitCode
+      : params.adUnitElementId || internal.autoDetectAdUnitElementId(adUnitCode);
+    const placement = (params.useAdUnitCodeAsPlacement === true) ? adUnitCode : params.placement;
```

Comment:
Hi @osazos , I don't see `useAdUnitCodeAsPlacement` and `useAdUnitCodeAsAdUnitElementId` params mentioned in the `adagioBidAdapter.md`, nor do I see them in the [Prebid.js docs repo,](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/adagio.md) I think you should describe them there. Also you should also cover these params with some unit test.